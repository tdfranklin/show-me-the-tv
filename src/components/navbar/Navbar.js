import React from 'react';
import AppBar from 'material-ui/AppBar';

const Navbar = (props) => <AppBar title='Show Me The TV!' showMenuIconButton={false} />;

export default Navbar;