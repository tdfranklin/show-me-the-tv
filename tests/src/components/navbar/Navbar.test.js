import React from 'react';
import ReactDOM from 'react-dom';
import Navbar from '../../../../src/components/navbar/Navbar';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('Navbar', () => {
    it('matches the snapshot', () => {
        const tree = shallow(<Navbar />);
        expect(toJson(tree)).toMatchSnapshot();
    });
});