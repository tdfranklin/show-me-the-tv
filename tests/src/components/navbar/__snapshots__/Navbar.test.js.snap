// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar matches the snapshot 1`] = `
<AppBar
  showMenuIconButton={false}
  title="Show Me The TV!"
  zDepth={1}
/>
`;
